services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/validation.py

```diff
@@ -125,27 +125,32 @@ def _gen_parts_env_vars(
     return [f"{k}={v}" for k, v in dict_spec_env_vars.items()]
 
 
-def _inject_backend_networking(
-    parsed_compose_spec: dict[str, Any], network_name: str = "__backend__"
-) -> None:
+DEFAULT_BACKEND_NETWORK_NAME = "__backend__"
```

Comment:
MINOR: this is internal, right? therefore enforce it as protected ``_DEFAULT_BACKEND_NETWORK_NAME``

Reply:
It's fine to make it private since it is used only here in my opinion as well.